src/main/java/fi/aalto/cs/apluscourses/utils/StringUtil.java

```diff
@@ -0,0 +1,25 @@
+package fi.aalto.cs.apluscourses.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringTokenizer;
+
+public class StringUtil {
+
+  private StringUtil() {}
+
+  /**
+   * Separates the given text into a String[] by tokenizing with the specified delimeter.
+   * @param phrase The String to be tokenized.
+   * @param delim The delimeter used in the tokenizing.
+   * @return The String[] containing the tokens.
+   */
+  public static String[] getArrayOfTokens(String phrase, char delim) {
```

Comment:
The difference between this and `String::split` is still a bit unclear to me but ok. :D